examples/step-17/step-17.cc

```diff
@@ -241,19 +251,20 @@ namespace Step17
 
   // @sect4{ElasticProblem::ElasticProblem}
 
-  // The first step in the actual implementation is the constructor
-  // of the main class. Apart from initializing the same member variables that
-  // we already had in step-8, we here initialize the MPI communicator
-  // variable we shall use with the global MPI communicator linking all
-  // processes together (in more complex applications, one could here use a
-  // communicator object that only links a subset of all processes), and call
-  // the Utilities helper functions to determine the number of processes and
-  // where the present one fits into this picture. In addition, we make sure
+  // The first step in the actual implementation is the constructor of
+  // the main class. Apart from initializing the same member variables
+  // that we already had in step-8, we here initialize the MPI
+  // communicator variable we shall use with the global MPI
+  // communicator linking all processes together (in more complex
+  // applications, one could here use a communicator object that only
+  // links a subset of all processes), and call the Utilities::MPI
+  // helper functions to determine the number of processes and where
+  // the present one fits into this picture. In addition, we make sure
   // that output is only generated by the (globally) first process. As
-  // <code>this_mpi_process</code> is determined after creation of pcout, we
-  // cannot set the condition through the constructor, i.e. by
-  // <code>pcout(std::cout, this_mpi_process==0)</code>, but set the
-  // condition separately.
+  // <code>this_mpi_process</code> is determined after creation of
+  // pcout, we cannot set the condition through the constructor,
+  // i.e. by <code>pcout(std::cout, this_mpi_process==0)</code>, but
+  // set the condition separately.
```

Comment:
Is there any reason why we cannot initialize `this_mpi_process` at the top of the constructor (and also move things around in the declaration) and then use

```cpp
pcout(std::cout, this_mpi_process == 0)
```

to get rid of this discussion?